.github/workflows/zips.yml

```diff
@@ -129,19 +132,61 @@ jobs:
         with:
```

Comment:
I can't comment on the specific part of the diff here (since it hasn't changed) but this still checks out the Wiki repo which is the most time consuming part of the process. This is of course no longer needed 😄 

<img width="757" alt="image" src="https://user-images.githubusercontent.com/1621608/232855163-e9600dde-5502-4ac5-8560-b8d925e3f38c.png">